packages/lesswrong/components/users/FollowUserButton.tsx

```diff
@@ -36,8 +37,17 @@ export const FollowUserButton = ({user, classes}: {
     onSubscribe ?? (() => {}),
   );
 
+  const [sendNewFollowDM] = useMutation(gql`
+    mutation sendEventTriggeredDM($eventType: String!) {
+      sendEventTriggeredDM(eventType: $eventType)
+    }
+  `, {
+    ignoreResults: true
+  });
+
   const handleSubscribe = (ev: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
     void toggleSubscribed(ev);
+    void sendNewFollowDM({variables: {eventType: "newFollowSubscription"}})
```

Comment:
Having this happen after voiding `toggleSubscribed` doesn't seem like it'll reliably cause the DM to be triggered, since it might hit the backend before the subscription gets created in the database

Reply:
(an additional problem is that we can't just await `toggleSubscribed`, since it's already the optimistic operation that doesn't wait for the backend response to assume success.)